### DISCUSSION TITLE
using Override on installs

### DISCUSSION THREAD

**Discussion by MBstatsara** [Romanitho/Winget-AutoUpdate#626] [General]
Hey
I seem to be missing the override option for new installs, i see its been removed from winget install and the only option is for mods, which is only working on updates?

How do we now run winget-install.ps1 with the override switch?

**Comment by Romanitho** [comment]
https://github.com/Romanitho/Winget-Install

**Reply by MBstatsara** [reply]
Thanks!
I thought it had been pushed into the WAU now

**Reply by Romanitho** [reply]
it is. You can use the same command line in theory
